### DISCUSSION TITLE
Game progress not being saved

### DISCUSSION THREAD

**Discussion by Dmgvol** [Kellegram/Ghostrunner_faq#5] [Bugs and glitches]
It's a popular issue where people complain about the game not saving game progress, you should add it to the faq.

It's most likely a third-party software blocking access, I would suggest adding GR to antivirus exceptions or/and running it as administrator as a quick test to see if the issue was resolved.
Or giving additional admin permissions to the save folder/file.

**Comment by Kellegram** [comment]
If anyone has any ideas, clues or information they can provide, we might be able to find a solution to this eventually or at least a workaround. People who have this issue can test launching the game as administrator to see if that helps. That being said, do not run the game as admi long-term, games should not be run as admin, only do this to test.

**Comment by Kellegram** [comment]
It seems like this might not be related to file permissions on Windows after all, as this issue has just occurred on Linux for someone.
